controllers/datamover.go

```
@@ -25,6 +28,29 @@ func (r *DPAReconciler) ReconcileDataMoverController(log logr.Logger) (bool, err
 		return false, err
 	}
 
+	// check volSync is installed/deployment exists to use data mover
+	if dpa.Spec.Features != nil && dpa.Spec.Features.EnableDataMover {
+
+		// create new client for deployments outside of adp namespace
+		kubeConf := config.GetConfigOrDie()
+
+		clientset, err := kubernetes.NewForConfig(kubeConf)
+		if err != nil {
+			return false, err
+		}
+
+		_, err = clientset.AppsV1().Deployments(common.VolSyncDeploymentNamespace).Get(context.TODO(), common.VolSyncDeploymentName, metav1.GetOptions{})
+		if err != nil {
+
+			if k8serror.IsNotFound(err) {
+				log.Info("volSync operator not found. Please install")
```

I would return an error here so that it will get put on the DPA status